libkbfs/backpressure_disk_limiter.go

```diff
@@ -846,28 +899,33 @@ func (bdl *backpressureDiskLimiter) getDelayLocked(
 	return time.Duration(delayScale * float64(maxDelay))
 }
 
-func (bdl *backpressureDiskLimiter) onBeforeBlockPutError(err error) (
+func (bdl *backpressureDiskLimiter) reserveError(err error) (
```

Comment:
I know this is a pre-existing condition, but do you think we need to take `bdl.lock` in this function, in order to get a consistent view of the bytes and files together?